eZ/Bundle/EzPublishCoreBundle/Resources/config/templating.yml

```diff
@@ -246,3 +246,17 @@ services:
     eZ\Publish\Core\MVC\Symfony\Templating\Twig\Extension\DataAttributesExtension:
         autoconfigure: true
         public: false
+
+    eZ\Bundle\EzPublishCoreBundle\Templating\Twig\ContextAwareTwigVariablesExtension:
+        arguments:
+            $configResolver: "@ezpublish.config.resolver"
+        tags:
+            - { name: twig.extension }
+
+    eZ\Publish\Core\MVC\Symfony\EventListener\ContentViewSubscriber: ~
+
+    eZ\Publish\Core\MVC\Symfony\View\GenericVariableProviderRegistry:
+        arguments:
+            $twigVariableProviders: !tagged ezplatform.view.parameter_provider
```

Comment:
Where is this tag configured?
If this is for 3rd party only, then $container->registerForAutoconfiguration() call in a bundle (or Extension, don't remember) would be nice
Moreover I'm still gonna insist on SPI layer consistency, since in this package it actually exists, e.g. `\eZ\Publish\SPI\MVC\View\VariableProvider` (I have no problem with the fact that this namespace doesn't exist yet, let's start to pay attention to those details).